### PR TITLE
Changing Zpages to Activity style

### LINKED ISSUE #817: Convert Zpages to Activity style
Convert ZPages to activity style
Add tests

### PR DESCRIPTION
**Pull request by eddynaka**
Fixes #817.

## Changes
- changing ZPagesExporter to Activity Style

For significant contributions please make sure you have completed the following items:

* [ ] Design discussion issue #
* [ ] Changes in public API reviewed

### CODE REVIEW

File: samples/Exporters/Console/TestZPagesExporter.cs [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+            using var openTelemetry = OpenTelemetrySdk.EnableOpenTelemetry(
+                builder => builder
+                    .AddActivitySource("zpages-test")
+                    .UseZPagesExporter(
+                        processorConfigure: processor => processor
+                            .AddProcessor((next) => zpagesProcessor)));
+
+            ActivitySource activitySource = new ActivitySource("zpages-test");
+
+            while (true)
```
Comment (cijothomas) [inline samples/Exporters/Console/TestZPagesExporter.cs:49]: this run forever?

Comment (eddynaka) [reply samples/Exporters/Console/TestZPagesExporter.cs:49]: At least, in the example we had the same: https://github.com/open-telemetry/opentelemetry-dotnet/blob/4eece94ec6712da7d7583e9bdc0b965ca90ac106/samples/Exporters/Console/TestZPages.cs#L47-L51

Comment (cijothomas) [reply samples/Exporters/Console/TestZPagesExporter.cs:49]: Okay got it.

File: test/OpenTelemetry.Exporter.ZPages.Tests/OpenTelemetry.Exporter.ZPages.Tests.csproj [hunk 2]
```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
```
Comment (CodeBlanch) [inline test/OpenTelemetry.Exporter.ZPages.Tests/OpenTelemetry.Exporter.ZPages.Tests.csproj:4]: Will `net5.0` wok with current CI?

Comment (eddynaka) [reply test/OpenTelemetry.Exporter.ZPages.Tests/OpenTelemetry.Exporter.ZPages.Tests.csproj:4]: not sure! 👍 i will update!

File: src/OpenTelemetry.Exporter.ZPages/Implementation/ZPagesActivities.cs [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+#if NET452
+using OpenTelemetry.Internal;
+#endif
+
+namespace OpenTelemetry.Exporter.ZPages.Implementation
+{
+    /// <summary>
+    /// Implements the ZPages Activity Queue which stores all the required activity information.
+    /// </summary>
+    public static class ZPagesActivities
```
Comment (CodeBlanch) [inline src/OpenTelemetry.Exporter.ZPages/Implementation/ZPagesActivities.cs:30]: Should this be `internal`? Going off of the "Implementation" namespace. Also, something about the name seems off. Not a big deal if it's internal. Not sure what would be better, `ZPagesActivityQueue` or `ZPagesActivityTracker`? Those aren't great either.

File: src/OpenTelemetry.Exporter.ZPages/Implementation/ZPagesInformation.cs [hunk 4]
```diff
@@ -15,32 +15,32 @@
...
-using OpenTelemetry.Trace.Export;
 
 namespace OpenTelemetry.Exporter.ZPages.Implementation
 {
     /// <summary>
-    /// Stores the span information aggregated according to span name.
+    /// Stores the activity information aggregated according to activity name.
     /// </summary>
-    public class ZPagesSpanInformation
+    public class ZPagesInformation
```
Comment (CodeBlanch) [inline src/OpenTelemetry.Exporter.ZPages/Implementation/ZPagesInformation.cs:28]: Same feedback on the `ZPagesInformation` name. I think it should also be `internal`. But would it also be better with a different name? `ZPagesActivityAggregate`? 🤷‍♂️

File: src/OpenTelemetry.Exporter.ZPages/Implementation/ZPagesActivities.cs (hunk 3 above)
Comment (eddynaka) [reply src/OpenTelemetry.Exporter.ZPages/Implementation/ZPagesActivities.cs:30]: i think tracker gets a better naming. queue it assumes that we have queues, but we have dictionaries. i'm updating!

**Pull request merged**